dashboard/src/pages/apps/[id]/settings.tsx

```diff
@@ -91,6 +93,25 @@ const SettingFieldSet = styled('div', {
     background: vars.bg.white1,
   },
 })
+const EnvVarContainerClass = style({
```

Comment:
[ask] これはこう書くと何か不都合がありました？
```suggestion
const EnvVarContainerClass = styled('form', {
```

Reply:
"key", "value"のラベルを入力欄と同じ幅で表示するために同じクラスを付与しているのですが、ラベル側にform要素使うのは不適切かなと思いこのようにしていました。

https://github.com/traPtitech/NeoShowcase/blob/b64b09050ded9abbbdc8346a02b76d52dd3cec9f/dashboard/src/pages/apps/%5Bid%5D/settings.tsx#L587-L590

ただ確かにちょっとわかりづらいですね...

Reply:
あ、普通に共通化として切り出していたのですね 今理解しました 🙇 
このままにしておくかは細かいところだと思うので任せます